cltk/prosody/old_norse/verse.py

```diff
@@ -589,3 +591,57 @@ def find_alliteration(self):
         :return:
         """
         return Metre.find_alliteration(self)
+
+
+class PoetryTools:
+    def __init__(self, basic=True):
+        if basic:
+            self.syllabifier = Syllabifier(language="old_norse_ipa")
+            self.tr = Transcriber(DIPHTHONGS_IPA, DIPHTHONGS_IPA_class, IPA_class, old_norse_rules)
+            self.tagger = POSTag('old_norse')
+        else:
+            self.syllabifier = None
+            self.phonetic_transcriber = None
+            self.tagger = None
+
+
+class PoeticWord:
+    def __init__(self, text):
+        self.text = text
+        self.syl = []
+        self.length = []
+        self.stress = []
+        self.ipa_transcription = []
+
+    def apply_poetic_tool(self, poetry_tools: PoetryTools):
+        """
+        Compute the phonetic transcription of the word with IPA representation
+        Compute the syllables of the word
+        Compute the lentgh of each syllable
+        Compute if a syllable is stress of noe
+        Compute the POS category the word is in
+
+        :param poetry_tools:
+        :return:
+        """
+        phonemes = poetry_tools.tr.text_to_phonemes(self.text)
+        self.syl = poetry_tools.syllabifier.syllabify_phonemes(phonemes)
+        for i, syllable in enumerate(self.syl):
+            self.ipa_transcription.append([])
+            syl_len = measure_old_norse_syllable(syllable).value
+            syl_stress = 1 if i == 0 else 0
+
+            self.length.append(syl_len)
+            self.stress.append(syl_stress)
+            for c in syllable:
+                self.ipa_transcription[i].append(c.ipar)
+
+    def print(self):
```

Comment:
I don't think this is very useful, to be honest, since you already have attributes like `helgar.text` exposed. Is there a specific use case that I am not thinking of?

Reply:
True!

Reply:
Done.